schemas/2019-11-01/Microsoft.Network.json

```diff
@@ -1732,6 +1738,119 @@
       ],
       "description": "Microsoft.Network/networkWatchers"
     },
+    "networkWatchers_connectionMonitors": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "Microsoft.Network/networkWatchers/connectionMonitors"
+          ]
+        },
+        "apiVersion": {
+          "type": "string",
+          "enum": [
+            "2019-11-01"
+          ]
+        },
+        "location": {
+          "type": "string",
+          "description": "Connection monitor location."
+        },
+        "tags": {
+          "oneOf": [
+            {
+              "type": "object",
+              "additionalProperties": {
+                "type": "string"
+              }
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ],
+          "description": "Connection monitor tags."
+        },
+        "properties": {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/ConnectionMonitorParameters"
+            },
+            {
+              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
```

Comment:
Please find+replace all occurrences of `https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression` with `https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression` - I suspect that's causing the CIT failures

Reply:
ok will check